engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -13,7 +13,7 @@ <h4 translate>Basic Information</h4>
         <span class="info-label" translate>Name</span>
         <span class="info-value"
               bst-edit-text="contentHost.name"
-              readonly="denied('edit_content_hosts', contentHost)"
+              readonly="denied('edit_hosts', contentHost)"
```

Comment:
Shouldn't all of the permissions in the entire PR be using `host` instead of `contentHost`?  What happens when you pass a content host with a different ID and it's evaluated as a host?